Xamarin.MacDev/IPhoneDeviceType.cs

```diff
@@ -46,6 +46,22 @@ public enum AppleDeviceFamily
 		IPad   = 2,
 		TV     = 3,
 		Watch  = 4,
+		IPod   = 5,
+	}
+
+	public static class AppleDeviceFamilyExtensions
+	{
+		public static string GetDeviceClass (this AppleDeviceFamily family)
+		{
+			switch (family) {
+				case AppleDeviceFamily.TV: return "tvOS";
```

Comment:
Why "tvOS" and not "tv"? Are those strings user-facing? Cause "iphone" could be "iPhone" if that's the case.